test/e2e/lib/async-base-container.js

```diff
@@ -56,7 +56,6 @@ export default class AsyncBaseContainer {
 		}
 		await this.waitForPage();
 		await this.checkForUnknownABTestKeys();
-		await this.checkForConsoleErrors();
```

Comment:
ChromeDriver provides a log stream, not a log store. This means the first time we read the logs they won't be available for future calls.

I removed any existing attempt to read the logs (which never worked in CircleCI/TeamCity afaik) other than the hook to store the logs.